diff -up matrix-commander-8.0.5/requirements.txt.omv matrix-commander-8.0.5/requirements.txt
--- matrix-commander-8.0.5/requirements.txt.omv	2025-06-17 14:12:03.000000000 -0400
+++ matrix-commander-8.0.5/requirements.txt	2025-06-20 22:53:59.363186439 -0400
@@ -1,9 +1,6 @@
 aiohttp
 aiofiles>=0.6.0
-argparse
-asyncio
 async-timeout # see Issue 161
-datetime
 emoji
 markdown
 matrix-nio[e2e]>=0.14.1
@@ -12,4 +9,4 @@ notify2
 Pillow
 python_magic
 pyxdg
-uuid
+
diff -up matrix-commander-8.0.5/setup.cfg.omv matrix-commander-8.0.5/setup.cfg
--- matrix-commander-8.0.5/setup.cfg.omv	2025-06-20 22:52:56.982813318 -0400
+++ matrix-commander-8.0.5/setup.cfg	2025-06-20 22:53:42.020968237 -0400
@@ -32,10 +32,7 @@ python_requires = >=3.11
 install_requires =
     aiohttp
     aiofiles>=0.6.0
-    argparse
-    asyncio
     async-timeout # see Issue 161
-    datetime
     emoji
     markdown
     matrix-nio[e2e]>=0.14.1
@@ -44,8 +41,6 @@ install_requires =
     Pillow
     python_magic
     pyxdg
-    uuid
-
 
 [options.package_data]
 # add docu if there is any inside the module(s)
